de.json

```diff
@@ -435,17 +435,17 @@
         "CreateNew.ParticleSystem" : "Partikelsystem",
 
         "CreateNew.Object" : "Objekt",
-        "CreateNew.Object.AvatarCreator" : "Avatar-Wizard",
+        "CreateNew.Object.AvatarCreator" : "Avatar-Assistent",
```

Comment:
I used in my commit "Avatar-Ersteller", translates literally to "Avatar Creator" (instead of "Avatar-Assistant")